packages/server/src/Server.ts

```diff
@@ -1,10 +1,21 @@
-import {GlobalAcceptMimesMiddleware, ServerLoader, ServerSettings} from "@tsed/common";
+import {
+  $log,
+  GlobalAcceptMimesMiddleware,
+  ServerLoader,
+  ServerSettings,
+  Configuration,
+} from "@tsed/common";
 import "@tsed/swagger";
 import * as bodyParser from "body-parser";
 import * as compress from "compression";
 import * as cookieParser from "cookie-parser";
 import * as methodOverride from "method-override";
 import * as path from "path";
+import * as dotenv from "dotenv";
+dotenv.config({ path: "./../config" });
```

Comment:
There is a reason to have this line between two imports ? ^^

Reply:
Done.